### ISSUE TITLE
how to run code in modelnet40_normal_resampled?no body can make it?can the author of aticle help us?

### ISSUE THREAD

**Issue by longmalongma** [charlesq34/pointnet2#120]
how to run code in modelnet40_normal_resampled?no body can make it?can the author of aticle help us?

**Comment by iwtbam** [comment]
so do I. I need help!!!

**Comment by sheshappanavar** [comment]
@longmalongma and @iwtbam please check https://github.com/charlesq34/pointnet2/issues/41#issuecomment-579547056

**Comment by sheshap** [comment]
1) Set FLAGS.normal = True in train files.

2) Replace 
l0_xyz = point_cloud
l0_points = None

by 
l0_xyz = tf.slice(point_cloud, [0,0,0], [-1,-1,3])
l0_points = tf.slice(point_cloud, [0,0,3], [-1,-1,3])

in model files.

**Comment by lyy-1997-hub** [comment]
> 1. 在训练文件中设置FLAGS.normal = True。
> 2. 替换
>    l0_xyz = point_cloud 
>    l0_points =无
> 
> 通过
> l0_xyz = tf.slice（point_cloud，[0,0,0]，[-1，-1,3]）
> l0_points = tf.slice（point_cloud，[0,0,3]，[-1，-1， 3]）
> 
> 在模型文件中。

I tried,but it didn't work.  :(

**Comment by sheshap** [comment]
https://github.com/charlesq34/pointnet2/issues/41

**Comment by lyy-1997-hub** [comment]
> #41

Yes!!! I succeeded yesterday through the method you said,and thanks again > <.But I feel little confused about 
"l0_xyz = tf.slice（point_cloud，[0,0,0]，[-1，-1,3]）
l0_points = tf.slice（point_cloud，[0,0,3]，[-1，-1， 3]）"
If you have time，could you please explain to me the function of these two lines of code？Thanks！

**Comment by lyy-1997-hub** [comment]
> #41

 I already understand about how the code works now.Thanks again!
